modules/exploits/windows/browser/example.rb

```diff
@@ -0,0 +1,147 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
```

Comment:
@wchen-r7 can you give us an up-to-date example of a browser exploit for AutoPwn2?